content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md

```diff
@@ -0,0 +1,29 @@
+---
+date: '2022-11-15'
+title: "Release Notes"
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-release-notes
+    weight: 6000
+section_menu: corda-5-beta
+---
+
+Corda 5.0 Beta is a pre-release version for testing purposes only.
+{{< note >}}
+If you are not part of the Beta program, the Corda 5.0 Beta documentation is for information only.
+R3 will be running a Beta Program for Corda 5.0 beginning in 2023. Contact R3 to register your interest.
```

Comment:
```suggestion
R3 will be running a Beta program for Corda 5.0 beginning in 2023. Contact R3 to register your interest.
```
Is it Beta Program or Beta program? (See the sentence above.)

Reply:
My take was if we're capitalising "beta" we're saying the "Beta Program" like it's the name of something. If we don't capitalise beta it's just describing the program.
![image](https://user-images.githubusercontent.com/103633799/209109563-89ff1b11-531f-4a10-a0f6-246fe05e2fe1.png)
Open to correction though.. @AnnaK-R3 @paulmoloneyr3 @colin-hillis

Reply:
Ok, I understand. In this case, please close this comment and apply the change for the sentence above.